tests/test_parts.py

```diff
@@ -63,25 +63,7 @@ def test_partconfig_bad_property():
     err = raised.value.errors()
     assert len(err) == 1
     assert err[0]["loc"] == ("color",)
-    assert err[0]["msg"] == "extra fields not permitted"
-
-
-def test_partconfig_bad_type():
```

Comment:
No longer necessary because the `source` configuration validator failing will prevent the `charm-requirements` validator from running.